src/App.css

```
@@ -137,4 +137,19 @@ input[type="number"] {
 
 .high-contrast.dark .text-gray-600 {
   color: #90a4ae;
-}
\ No newline at end of file
+}
+/* Digest feed staggered reveal */
+@keyframes fadeSlideIn {
```

Stylelint (17.13.0)

[error] Expected keyframe name "fadeSlideIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)